### ISSUE TITLE
Limit the child pages that can be created

### ISSUE THREAD

**Issue by tbrlpld** [MozillaFoundation/foundation.mozilla.org#10254]
Currently, when a page does not limit which pages can be used as its child pages editors are presented with the option to practically add any page type. 

![](https://user-images.githubusercontent.com/24797493/222755347-a8fe3457-8f48-466b-afe6-3bed25d14923.png)

This is overwhelming to the editor and makes it hard to find the proper page type to use. 

Additionally, it allows structures that are non-sensical. E.g. creating a homepage instance somewhere in the page tree. 

We should review our page types and make sure the structures that can be created make sense. 

There are two ways the page structure can be limited.

1. A page type can define its allowed [child page types](https://docs.wagtail.org/en/stable/reference/pages/model_reference.html#wagtail.models.Page.subpage_types)
1. A page type can define its allowed [parent page types](https://docs.wagtail.org/en/stable/reference/pages/model_reference.html#wagtail.models.Page.parent_page_types)

I think it might make sense to start with limiting the parent page types first because this will remove the page from being added as a child to any other child. 
Child restrictions only limit which pages can be added to that specific page type. 

E.g. adding the restriction to the HomePage to be only created at the root page removes the need to exclude the HomePage from other page type's child lists.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/TP1-137)

**Comment by tbrlpld** [comment]
@kristinashu Also interested if there any thoughts on this from design perspective as you have more insights into stakeholder requirements and content strategy.

**Comment by kristinashu** [comment]
Yes, simplifying would be amazing and I had made a spreadsheet about this - [third tab called "Page types" ](https://docs.google.com/spreadsheets/d/1AwLF6dw05Jn2WZu-xvxUgm8VQ4udriZW6d_NfdVI9ns/edit?usp=sharing).

I first row (heading with the black background) are the parent page types and below that are all the possible child page types. I started to put checkmarks to show the current state and I've made some comments about which ones I think should be removed. I would love input on it!

<img width="1429" alt="image" src="https://user-images.githubusercontent.com/1068783/222759441-f346d143-4bd2-455c-8fbc-b6693bec7d41.png">
